tests/test_wps_generic_shape_properties.py

```diff
@@ -30,20 +129,21 @@ def test_simple(self):
         props = json.loads(out['properties'])
         assert {'centroid', 'area', 'perimeter', 'gravelius'}.issubset(props[0].keys())
 
-        np.testing.assert_approx_equal(props[0]['area'], 44877188052)
-        np.testing.assert_approx_equal(props[0]['centroid'][0], -72.69128332)
-        np.testing.assert_approx_equal(props[0]['centroid'][1], 49.50119363)
-        np.testing.assert_approx_equal(props[0]['perimeter'], 1861151.69970881)
-        np.testing.assert_approx_equal(props[0]['gravelius'], 2.4783578)
+        # np.testing.assert_approx_equal(props[0]['area'], 24804229364.)
+        # np.testing.assert_approx_equal(props[0]['centroid'][0], -73.58396746)
+        # np.testing.assert_approx_equal(props[0]['centroid'][1], 46.88002938)
+        # np.testing.assert_approx_equal(props[0]['perimeter'], 1520538.80767662)
+        # np.testing.assert_approx_equal(props[0]['gravelius'], 2.7235145984)
```

Comment:
But then we're not checking the actual results. My suggestion is to use a very simple shape (square), whose properties are obvious, so we can compare the outcomes with expectations.